docs/sql-ref-literals.md

```diff
@@ -275,6 +275,10 @@ E [ + | - ] digit [ ... ]
 
     Case insensitive, indicates `DECIMAL`, with the total number of digits as precision and the number of digits to right of decimal point as scale.
 
+* **default (no postfix)**
+
+    Indicates `DECIMAL`, same as the `BD` postfix.
```

Comment:
exponential w/o postfix, such as 1.2E3, is a double by default

Reply:
Ah, ok. I can make another PR to clarify this point, unless you prefer to do it yourself.

Reply:
Ohh actually let me revert this first. Let's make the proper fix together.

Reply:
Sure, if you prefer to do that. I will make a new PR to clarify the default type with and without the exponent.